bee-message/src/payload/data.rs

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use bee_packable::{error::{PackPrefixError, UnpackPrefixError}, Packable, Packer, PackError, Unpacker, UnpackError, VecPrefix};
+
+use core::{fmt, convert::Infallible};
```

Comment:
this is very VERY nitpicky but you've been using `core::fmt::{Display|Debug}` everywhere else.

Reply:
Yeah I haven't done a style check yet 😅